tests/test_existing_excerpt.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import time
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.keys import Keys
+
+@pytest.fixture
+def prerequisite(base_url, selenium):
+    # create new excerpt only for this testing
+    selenium.get('{0}/admin/login/'.format(base_url))
+    selenium.maximize_window()
+
+    # login as admin
+    username = selenium.find_element_by_id('id_username')
+    password = selenium.find_element_by_id('id_password')
+    login = selenium.find_element_by_class_name('submit-row')
+    username.send_keys("admin")
+    password.send_keys("admin")
+    login.send_keys(Keys.RETURN)
+
+    selenium.get('{0}/'.format(base_url))
+
+    # go to new excerpt menu
+    new_excerpt = selenium.find_element_by_link_text('⌗ New excerpt')
+    new_excerpt.send_keys(Keys.RETURN)
+
+    # insert excerpt name
+    excerpt_name = selenium.find_element_by_id('id_name')
+    excerpt_name.send_keys('existing_excerpt')
+
+    # choose an area in monaco (North = 43.734716500825 | East = 7.42564201354981 | South = 43.7289719167851 | West = 7.41568565368652)
+    north = selenium.find_element_by_id('id_north')
+    north.clear()
+    north.send_keys("43.734716500825")
+    east = selenium.find_element_by_id('id_east')
+    east.clear()
+    east.send_keys("7.42564201354981")
+    south = selenium.find_element_by_id('id_south')
+    south.clear()
+    south.send_keys("43.7289719167851")
+    west = selenium.find_element_by_id('id_west')
+    west.clear()
+    west.send_keys("7.41568565368652")
+
+    # choose the file format
+    formats = selenium.find_element_by_id('id_formats_3')
+    formats.send_keys(Keys.RETURN)
+
+    # submit
+    create = selenium.find_element_by_name('submit')
+    create.send_keys(Keys.RETURN)
+    time.sleep(60)
+
+    return prerequisite
```

Comment:
This will return the function object, which doesn't make a lot of sense. Can probably be simply removed. (Not all fixtures have to return something.)